### PR TITLE
web/satellite: Added token transaction history to new payment methods flow

### PR DESCRIPTION
**Pull request by cl-mitch**
What: Added code for SortingHeader2.vue, VerticalArrows.vue, and TokenTransactionItem.vue. This code is for some of the stubbed components on PaymentMethodsTab.vue.

Why:

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/components/account/billing/depositAndBillingHistory/SortingHeader2.vue [hunk 1]
```diff
@@ -2,12 +2,151 @@
...
 export default class SortingHeader2 extends Vue {
+
+    public DATE_DIRECTION = 'date-descending'
+    public AMOUNT_DIRECTION = 'amount-descending'
+    public STATUS_DIRECTION = 'status-descending'
+
+    public dateSortDirection: SortDirection = SortDirection.ASCENDING;
+    public amountSortDirection: SortDirection = SortDirection.ASCENDING;
+    public statusSortDirection: SortDirection = SortDirection.ASCENDING;
+
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/account/billing/depositAndBillingHistory/SortingHeader2.vue:68]: extra lines

File: web/satellite/src/components/account/billing/depositAndBillingHistory/SortingHeader2.vue [hunk 2]
```diff
@@ -2,12 +2,151 @@
...
 </template>
 
 <script lang="ts">
-import { Vue } from 'vue-property-decorator';
+import { Component, Vue } from 'vue-property-decorator';
 
+import VerticalArrows from '@/components/common/VerticalArrows.vue';
+
+import { SortDirection } from '@/types/common';
+
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/account/billing/depositAndBillingHistory/SortingHeader2.vue:51]: extra line

File: web/satellite/src/components/account/billing/paymentMethods/TokenTransactionItem.vue [hunk 3]
```diff
@@ -1,13 +1,150 @@
...
 </template>
 
 <script lang="ts">
-import { Vue } from 'vue-property-decorator';
+import { Component, Prop, Vue } from 'vue-property-decorator';
 
+import { PaymentsHistoryItem } from '@/types/payments';
+
+// @vue/component
+@Component({
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/account/billing/paymentMethods/TokenTransactionItem.vue:45]: delete redundant decorator body

Comment (cl-mitch) [reply web/satellite/src/components/account/billing/paymentMethods/TokenTransactionItem.vue]: removed

File: web/satellite/src/components/account/billing/depositAndBillingHistory/SortingHeader2.vue (hunk 2 above)
Comment (cl-mitch) [reply web/satellite/src/components/account/billing/depositAndBillingHistory/SortingHeader2.vue:51]: removed

File: web/satellite/src/components/account/billing/paymentMethods/TokenTransactionItem.vue [hunk 4]
```diff
@@ -1,13 +1,146 @@
...
+    </div>
 </template>
 
 <script lang="ts">
-import { Vue } from 'vue-property-decorator';
+import { Prop, Vue } from 'vue-property-decorator';
 
+import { PaymentsHistoryItem } from '@/types/payments';
+
+// @vue/component
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/account/billing/paymentMethods/TokenTransactionItem.vue:44]: I thought that we are leaving just @Component. Not sure if it is working like this. Please test it. If so, LGTM

Comment (cl-mitch) [reply web/satellite/src/components/account/billing/paymentMethods/TokenTransactionItem.vue:44]: Hi Nikolai, I somehow missed this comment, but adding that `@Component` resolved a long standing issue for me! Thanks!

**Review by mobyvb — APPROVED** [review]

**Review by cam-a — APPROVED** [review]

**Comment by cl-mitch** [comment]
run jenkins

**Pull request merged**